course-preparation/Ansible/group_vars/mjarfadhi.YML

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Mohamed A. Ali                      # Real name of the person
-  org: SomlaiREN                                # Organisation/team they work in
+  org: somlaiREN                                # Organisation/team they work in
   	username: mjarfadhi                     # username on the lab machine
   	github: mjarfadhi                       # github user account
   	git:                                     # git configuration on the lab machines
```

Comment:
The rest of the git variables are missing here - 
something like : 
```
 git:                                     # git configuration on the lab machines
      user:
        name: Bruce Becker
        email: brucellino@gmail.com
      core:
        editor:
          name: vim
          package:
            RedHat: vim
            Debian: vim
```

(change your name and email address)